ietf/templates/doc/document_draft.html

```diff
@@ -658,45 +658,24 @@
                 </i>
                 Nits
             </a>
-            <div class="dropdown inline">
-                <button class="btn btn-primary btn-sm dropdown-toggle"
-                        type="button"
-                        id="ddSearchMenu"
-                        data-bs-toggle="dropdown"
-                        aria-expanded="true">
+            <a class="btn btn-primary btn-sm"
+               href="https://mailarchive.ietf.org/arch/search/?q=%22{{ doc.name }}%22"
+               rel="nofollow"
+               target="_blank">
+                <i class="bi bi-search">
+                </i>
+                Search email archive
+            </a>
+            {% if user|has_role:"Area Director" %}
+                <a class="btn btn-primary btn-sm"
+                   href="https://www.iesg.org/bin/c5i?mid=6&amp;rid=77&amp;target={{ doc.name }}"
```

Comment:
Note: In my limited testing, this always gets a 404 from `www.iesg.org`.

Reply:
Remove it - the IESG no longer uses ARO.